src/examples/DataCenter/Input/paraCfgCurrent.txt

```diff
@@ -0,0 +1,20 @@
+ParameterName	ParameterValue
+environmentFileName	DataCenter.xml
+generalizedScenario	1
+indoorSwitch	1
+mobilitySwitch	0
+mobilityType	1
+numberOfNodes	2
+numberOfTimeDivisions	15
+referencePoint	[3,3,2]
+selectPlanesByDist	0
+switchmodelRray	1
```

Comment:
not part of the current code

Reply:
oops sorry I forgot to change this. Thanks for pointing this out